google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/DirectExecuteResultSet.java

```diff
@@ -180,6 +180,12 @@ public long getLong(String columnName) {
     return delegate.getLong(columnName);
   }
 
+  @Override
```

Comment:
@olavloite Could you give this a look if these are all the changes that are needed in the Connection API or do we need to do more changes to support this new type?

Reply:
The rest of the changes LGTM.